src/components/message-pane/message-log-area/styles.module.scss

```
@@ -50,6 +50,34 @@
   }
 }
 
+.reactSuggestionList {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 3px;
+  gap: 6px;
+
+  -webkit-user-select: none;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)